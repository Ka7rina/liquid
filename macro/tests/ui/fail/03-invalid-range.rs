use liquid_macro::seq;

seq!(N in a..b {
    // nothing
});

fn main() {}
