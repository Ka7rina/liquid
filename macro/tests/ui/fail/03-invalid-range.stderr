error: expected unsuffixed integer literal
 --> $DIR/03-invalid-range.rs:3:11
  |
3 | seq!(N in a..b {
  |           ^
